vyper/builtin_functions/convert.py

```diff
@@ -461,6 +462,18 @@ def to_bytes(expr, arg, out_typ):
     return IRnode.from_list(arg, typ=out_typ)
 
 
+@_input_types("int")
+def to_enum(expr, arg, out_typ):
+    if not is_base_type(arg.typ, "uint256"):
+        _FAIL(arg.typ, out_typ, expr)
+
+    _check_bytes(expr, arg, out_typ, 32)
+    arg = _int_to_int(arg, out_typ)
+
+    clamped_arg = int_clamp(arg, len(out_typ.members), False)
```

Comment:
can you use the keyword arguments here?
```suggestion
    clamped_arg = int_clamp(arg, bits=len(out_typ.members), signed=False)
```